### PR TITLE
Update tanks tutorial

### PR DESCRIPTION
**Pull request by MasterPtato**
Fixes WEB-128

### CODE REVIEW

File: src/pages/learn/godot/tutorials/bomber-demo/index.mdx
Comment (AngelOnFira) [inline src/pages/learn/godot/tutorials/bomber-demo/index.mdx:1]: Ty for the fixes on this file, good to see some items I didn't consider 🙌

**Review by AngelOnFira — APPROVED** [review]

File: src/pages/learn/html5/tutorials/crash-course.mdx [hunk 1]
```diff
@@ -46,8 +46,6 @@ You'll need a `Dockerfile` to build and run your game server.
 
 <UnfamiliarWithDockerfiles />
 
----
-
 ## Step 2: Integrate Rivet Matchmaker
 
 ### Install the Rivet library
```
Comment (NathanFlurry) [inline src/pages/learn/html5/tutorials/crash-course.mdx:51]: <i class='graphite__hidden'>[Re: line 55]<br/><br/></i>

Remove version<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rivet-gg/site/129?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Review by NathanFlurry — CHANGES_REQUESTED** [review]
we should be using npm in our tutorials because a lot of people (esp newer to js) are not familiar with yarn. or the super hip js devs don't like yarn anymore. i don't remember if the example is using yarn or npm.

File: src/pages/learn/html5/tutorials/tanks-canvas-socketio.mdx [hunk 2]
```diff
@@ -17,83 +17,56 @@ import { TutorialVideo } from '@/components/TutorialVideo';
...
-        --matchmaker-port 3000 ^
-        --matchmaker-dockerfile Dockerfile ^
-        --cdn ^
-        --cdn-build-command "npm install && npm run build:client:prod" ^
-        --cdn-build-output ./dist/
-    ```
-  </code>
-</CodeGroup>
+```bash
+rivet init --html5
```
[the remark this answers is not in the document]
Comment (MasterPtato) [reply src/pages/learn/html5/tutorials/tanks-canvas-socketio.mdx:57]: yes, its already in the repo

File: src/components/mdx.jsx [hunk 3]
```diff
@@ -136,8 +137,9 @@ export function EnvTokenServer() {
   return (
     <Info>
       <p>
-        In development, `RIVET_TOKEN` will use the development token generated by `rivet init`. In production,
-        `RIVET_TOKEN` is automatically added to your environment by Rivet.
+        In development, <code>RIVET_TOKEN</code> will use the development token generated by{' '}
+        <code>rivet run</code>. In production,&nbsp;<code>RIVET_TOKEN</code> is automatically added to your
+        environment by Rivet.
```
[the remark this answers is not in the document]
Comment (MasterPtato) [reply src/components/mdx.jsx:142]: Could you explain how it automatically gets added by the CDN?

File: src/pages/learn/html5/tutorials/tanks-canvas-socketio.mdx [hunk 4]
```diff
@@ -157,20 +126,31 @@ async function setupConnection(socket: Socket) {
...
 ```
 
-<Accordion title='What did this do?'>
-  - Build & upload your Docker image for [Rivet Dynamic Servers](/docs/dynamic-servers/index) - Build &
-  upload your site for [Rivet CDN](/docs/cdn) - Create a version on Rivet - Deploy the version to the
-  *Production* namespace
+<Accordion title='What does this do?'>
+  - Builds & uploads your Docker image for [Rivet Dynamic Servers](/docs/dynamic-servers/index)
+  - Builds & uploads your site for [Rivet CDN](/docs/cdn)
+  - Creats a version on Rivet - Deploys the version to the *Production* namespace
```
[the remark this answers is not in the document]
Comment (MasterPtato) [reply src/pages/learn/html5/tutorials/tanks-canvas-socketio.mdx:153]: We don't have any docs on namespaces

File: src/components/mdx.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (NathanFlurry) [reply src/components/mdx.jsx:140]: https://b5a845b5.site-ckd.pages.dev/docs/general/concepts/cli-tips#build-command-environment-variables

we can restack on top of this or add a dedicated docs page later (as a ticket)

**Review by NathanFlurry — CHANGES_REQUESTED** [review]

File: src/components/mdx.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (MasterPtato) [reply src/components/mdx.jsx:140]: I don't see how that explains how the CDN is adding the env vars. Do you mean the CLI?

**Review by NathanFlurry — APPROVED** [review]

**Pull request merged**